src/common/utils/index.js

```diff
@@ -3,3 +3,103 @@ export const idExtractor = item => item.id;
 export const underscoreIdExtractor = item => item._id;
 
 export const isIdForCurrentUser = (id, currentUser) => currentUser.id === id;
+
+/**
+ * Get the Best Grid Spacing from the given itemWidths.
+ * The best spacing is the one with a margin that is close to minMargin.
+ *
+ * @param {Array} itemWidths An array of widths. I.e [10, 20, 30]
+ * @param {Int} availableWidth The width of the grid/The amount of space we can work with.
+ * @param {Int} minMargin The minimum margin to keep between items and around the edges.
+ * @returns {Object} An object containing the best spacing and the item width associated with it.
+ */
```

Comment:
💯 we should have these everywhere ™️ . It's like browsing Android's core classes